redisinsight/ui/src/components/cli/components/cli-search/CliSearchFilter/CliSearchFilter.tsx

```diff
@@ -35,17 +38,22 @@ const CliSearchFilter = ({ submitFilter, isLoading }: Props) => {
     setTypeSelected('')
   }, [matchedCommand])
 
-  const options: EuiSuperSelectOption<string>[] = FILTER_GROUP_TYPE_OPTIONS.map(
+  const groupOptions = [...commandGroups].sort().map((group: string) => ({
+    text: (GROUP_TYPES_DISPLAY as any)[group] || group.replace(/_/, ' '),
```

Comment:
I think better use `.replaceAll()`, or add flag `/g` to regexp

Reply:
yes, thanks! Done